css/style.css

```diff
@@ -64,55 +68,57 @@ h2 {
 .main-text {
   width: 100%;
   padding: 20px;
-  padding-right: 100px;
+  height: 100px;
   text-align: justify;
   font-family: 'Raleway', 'Helvetica';
 }
 
 .wrapper {
-  max-width: 960px;
+  max-width: 1150px;
   margin: 0px auto;
+  /*background-color: #e3f2f9;*/
 }
 
 
-/*Media accessibility stylings*/
+input[type=text] {
+  font-family: 'Raleway', 'Helvetica';
+  padding: 5px;
+  margin-top: 20px;
+}
 
+input[type=email] {
+  font-family: 'Raleway', 'Helvetica';
+  padding: 5px;
+  margin-top: 20px;
+}
 
-@media (max-width: 768px) {
-    .wrapper {
-        max-width: 768px;
-        margin: 0px auto;
-    }
+.dropDown {
+  margin-top: 20px;
 
-    /*.navigation {
-      width: 100%;
+}
 
-    }
+.submissionField {
+  width: 300px;
+  height: 100px;
+}
 
-    .list {
-      font-size: 15px;
-      font-family: 'Raleway', 'Helvetica';
-      line-height: 200%;
-      width: 50%;
-}*/
+/*Media accessibility stylings*/
 
 
-    .main-text {
-      width: 100%;
-      padding: 20px;
-      text-align: justify;
+@media (max-width: 768px) {
+    .wrapper {
+        max-width: 768px;
+        margin: 0px auto;
     }
 
     h1  {
        top: 10px;
        font-size: 30px;
-
     }
 
     h2  {
       top: 50px;
       font-size: 50px;
-
     }
 
   }
```

Comment:
Good job! Keep up the good work. High level comments:

 * Use vertical whitespace in your body copy
 * Give your links some sort of styling so they're obviously different than body copy
 * Try adding some styling to the dropdown and buttons so they don't stick out as much.